contrib/basicauth/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+nginx:
+  build: "nginx"
+  ports:
+    - "5441:5441"
+  links:
+    - registryv2:registryv2
+registryv2:
+  build: "../../"
+  ports:
+    - "5000"
```

Comment:
Should this be `- "5000:5000"`? Otherwise port 5000 is published to a random port on the host. If it doesn't have to be published to the host (only used for the private container network), this may not be needed at all